crates/bevy_pbr/src/render/light.rs

```diff
@@ -559,8 +559,10 @@ pub fn extract_lights(
         // NOTE: When using various PCF kernel sizes, this will need to be adjusted, according to:
         // https://catlikecoding.com/unity/tutorials/custom-srp/directional-shadows/
         let directional_light_texel_size = transform.radius_vec3a(Vec3A::new(
-            directional_light.shadow_projection.right - directional_light.shadow_projection.left,
-            directional_light.shadow_projection.top - directional_light.shadow_projection.bottom,
+            directional_light.shadow_projection.area.max.x
+                - directional_light.shadow_projection.area.min.x,
+            directional_light.shadow_projection.area.max.y
+                - directional_light.shadow_projection.area.min.y,
             0.,
         )) / directional_light_shadow_map.size as f32;
```

Comment:
This could be
```rs
        let directional_light_texel_size = transform.radius_vec3a(
            directional_light
                .shadow_projection
                .area
                .size()
                .extend(0.0)
                .into(),
        ) / directional_light_shadow_map.size as f32;
 ```